src/metabase/integrations/slack.clj

```diff
@@ -36,14 +36,17 @@
   (when-not (str/blank? channel-name)
     (if (str/starts-with? channel-name "#") (subs channel-name 1) channel-name)))
 
+(defsetting slack-cache
+  "A cache shared between instances for storing an instance's slack channels and users."
+  :visibility :internal
+  :type :json)
+
 (defsetting slack-files-channel
   (deferred-tru "The name of the channel to which Metabase files should be initially uploaded")
   :default "metabase_files"
   :setter (fn [channel-name]
             (setting/set-value-of-type! :string :slack-files-channel (process-files-channel-name channel-name))))
 
-(def ^:private ^String slack-api-base-url "https://slack.com/api")
-
```

Comment:
What's the reason for removing the constant and replacing it with magic strings below?

Reply:
This was only used in one place, so the abstraction about what `slack-api-base-url` is doesn't add much value. Now, readers of `do-slack-request` don't need to pop up and check what `slack-api-base-url`'s value is.

Reply:
Before, it was marked private since it is un-needed in other namespaces. Now its level of privacy is higher, since it's un-usable from anywhere besides `do-slack-request`.